.github/first-timers.yml

```diff
@@ -1,12 +1,22 @@
-# You can change the labels to suit your needs if "first-timers-only" is not what you are looking for.
-# These are some examples.
-labels:
-  - first-timers-only
-  - help-wanted
-  
-#If you would like to add your own template for the issue, add an .md file to your .github folder
-template: .github/first-timers-issue-template.md
-  
-# You can create the issue in a different repo than where the problem is. Just make sure you installed the bot on the configured repository.
-# The issue will link back to the original repository where the contribution will be made.
-#repository: repo-name 
+@@ -0,0 +1,21 @@
++# Configuration for welcome - https://github.com/behaviorbot/welcome
```

Comment:
Ah - i think this needs to be added to a new file called `config.yml` instead of to the first-timers.yml file -- can you change that, do you think? Thank you!!!!